projects/common/src/telemetry/providers/rudderstack/rudderstack-provider.ts

```diff
@@ -23,8 +26,16 @@ export class RudderStackTelemetry implements UserTelemetryProvider<RudderStackCo
     }
   }
 
-  public identify(userTraits: UserTraits): void {
-    identify(undefined, userTraits as apiObject);
+  public identify(): void {
+    this.http.get<UserTraits>('/user-info').subscribe(
```

Comment:
Should this have an accompanying unsubscribe statement somewhere?

Reply:
Didn't add the `unsubscribe` as this is a one-off call which will occur once precisely during the whole session.

Reply:
@cseas On second thoughts, not doing it for the http calls as this is taken care of internally
[Reference 1](https://lukaonik.medium.com/do-we-need-to-unsubscribe-http-client-in-angular-86d781522b99)
[Reference 2](https://stackoverflow.com/questions/35042929/is-it-necessary-to-unsubscribe-from-observables-created-by-http-methods)